opm/autodiff/StandardWell_impl.hpp

```diff
@@ -2229,4 +2232,88 @@ namespace Opm
             }
         }
     }
+
+
+
+
+    template<typename TypeTag>
+    double
+    StandardWell<TypeTag>::
+    relaxationFactorFraction(const double old_value,
```

Comment:
I think this method could be written quite a bit simpler:
```
const double possible_updated_value = old_value - dx;
if (possible_updated_value < 0.0) {
    return std::abs(old_value / dx) * 0.95;
} else if (possible_updated_value > 1.0) {
    return std::abs((1. - old_value) / dx) * 0.95;
} else {
    return 1.0;
}
```

Reply:
Mostly correct. I would like to add a few `assert` inside. Will update following suggestion.